steampy/confirmation.py

```diff
@@ -26,20 +27,28 @@ class Tag(enum.Enum):
 class ConfirmationExecutor:
     CONF_URL = "https://steamcommunity.com/mobileconf"
 
-    def __init__(self, trade_offer_id: str, identity_secret: str, my_steam_id: str, session: requests.Session) -> None:
-        self._trade_offer_id = trade_offer_id
+    def __init__(self, identity_secret: str, my_steam_id: str, session: requests.Session) -> None:
         self._my_steam_id = my_steam_id
         self._identity_secret = identity_secret
         self._session = session
+        self._confirmation = None
 
-    def send_trade_allow_request(self) -> dict:
+    def send_trade_allow_request(self, trade_offer_id: str) -> dict:
         confirmations = self._get_confirmations()
-        confirmation = self._select_trade_offer_confirmation(confirmations)
+        self._confirmation = self._select_trade_offer_confirmation(confirmations, trade_offer_id)
+        return self._send_confirmation()
+
+    def confirm_sell_listing(self, asset_id: str) -> dict:
+        confirmations = self._get_confirmations()
+        self._confirmation = self._select_sell_listing_confirmation(confirmations, asset_id)
+        return self._send_confirmation()
+
```

Comment:
I'm not sure how to call this method because I can't get the "send_trade_allow_request" name